_infra/helm/response-operations-ui/values.yaml

```diff
@@ -7,6 +7,7 @@ sendEmailToGovNotify: false
 admin:
   createAccountEmailExpiry: "2419200"
   updateAccountEmailExpiry: "259200"
+  completeToNotStartedWaitTime: "172800"
```

Comment:
Sorry hadn't noticed this, naturally if I deploy this into sandbox/dev it will pick this up, only locally will it be 5 minutes through config, does that matter, not sure, and why approving

Reply:
Updated to be shorter when in sandbox and dev